waspc/src/Wasp/Lib.hs

```diff
@@ -173,3 +180,16 @@ compileAndRenderDockerfile waspDir compileOptions = do
     Right appSpec -> do
       dockerfileOrGeneratorErrors <- DockerGenerator.compileAndRenderDockerfile appSpec
       return $ left (map show . toList) dockerfileOrGeneratorErrors
+
+deploy :: FilePath -> Path' Abs (Dir WaspProjectDir) -> [String] -> IO ()
```

Comment:
Since this is pretty complex stuff, maybe you can put a small comment explaining what this function does?